lib/resourceLog.ts

```diff
@@ -0,0 +1,60 @@
+import { ToolingInterface } from "../drivers/instrumentedResource";
+import { logger } from "./logger";
+import { writeFileSync } from "jsonfile";
+import { promises as fs } from "fs";
+
+
+abstract class ResourceLog {
+  protected readonly logger;
+  protected readonly entries: ToolingInterface[];
+  protected constructor(entries: ToolingInterface[]) {
+    this.entries = entries;
+    this.logger = logger;
+  }
+
+  abstract process(): void;
+}
+
+export class ResourceLogConsole extends ResourceLog {
+  constructor(entries: ToolingInterface[]) {
+    super(entries);
+  }
+  process(): void {
+    const header = `${'ACCOUNT_ID'.padEnd(16)}${'REGION'.padEnd(20)}${'TYPE'.padEnd(20)} ${'ID'.padEnd(40)} ${'STATE'}`;
+    const lines = this.entries.map((r) => `${(r.accountId || '').padEnd(16)}${(r.region || '').padEnd(20)}${(r.resourceType || '').padEnd(20)} ${r.resourceId.padEnd(40)} ${r.resourceState}`)
```

Comment:
Can we do this (and the other similar lines) in a slightly more structured way, that's easier to view and edit.  Maybe a list of string+pad, then a loop/function that pads the values and applies the separators.  If the fields are the same between all the different ResourceLog, then maybe the field building could be in the base-class?